app/common/services/strataService.js

```diff
@@ -197,6 +197,25 @@ angular.module('RedhatAccess.common').factory('strataService', [
                     return deferred.promise;
                 }
             },
+            articles: {
+                get: function (uri) {
+                    var deferred = $q.defer();
+                    var splitUri = uri.split('/');
+                    uri = splitUri[splitUri.length - 1];
```

Comment:
Still, seems rather hackish for no good reason. I personally would rewrite this, but let's wait for opinions from others :).